f3/granite.go

```diff
@@ -709,11 +695,16 @@ func (i *instance) tryCommit(round uint64) error {
 func (i *instance) beginDecide(round uint64) {
 	i.phase = DECIDE_PHASE
 	roundState := i.roundState(round)
-	if !roundState.committed.HasStrongQuorumAgreement(i.value.Head().CID) {
+	var (
+		signers    bitfield.BitField
+		signatures [][]byte
+		ok         bool
+	)
```

Comment:
I prefer not having to write (and delete) 3 times var, but I do not have a big preference. I guess I thought it is the idiomatic way in Go (since `var (...)` can only be used in cases like this one).